src/main/java/build/buildfarm/worker/Executor.java

```diff
@@ -46,10 +46,16 @@
   }
 
   private void runInterruptible() throws InterruptedException {
-    Command command;
-    try {
-      command = Command.parseFrom(workerContext.getBlob(operationContext.action.getCommandDigest()));
-    } catch (InvalidProtocolBufferException ex) {
+    ByteString commandBlob = workerContext.getBlob(operationContext.action.getCommandDigest());
+    Command command = null;
+    if (commandBlob != null) {
+      try {
+        command = Command.parseFrom(commandBlob);
+      } catch (InvalidProtocolBufferException ex) {
+      }
+    }
+
+    if (command == null) {
```

Comment:
Is there a way to pass the error upstream? Ideally with the missing digest. Ditto with the action.

Reply:
Currently, the path for this goes like this: The error stage drops the operationContext on the floor, the requeuer on the server notices the lack of updates and queues the operation again (without a check for validity, adding that in an update), and the cycle repeats. There aren't any great channels available for this, except for the server logs, which I'm woefully behind in terms of supplying any content for. Were you thinking of something more structured? Otherwise I'd like to hold off on this until I have put better logging into place.